NetCoreRestApi/DataLayer/UnitOfWorks/IUnitOfWork.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Linq;
+
+namespace DataLayer.UnitOfWorks
+{
+    public interface IUnitOfWork<T>
+    {
+        T GetById(int id);
+        IQueryable<T> GetAll();
+        IQueryable<T> Create(T entity);
+        IQueryable<T> Update(T entity);
+        IQueryable<T> Delete(int id);
```

Comment:
1 Use ICollection entities instead of T entity and int id
2. Update and Delete should return void

Reply:
done